### ISSUE TITLE
Pulling from multiple private repositories (on GitHub)

### ISSUE THREAD

**Issue by jacopo-chevallard** [Shippable/support#2982]
During CI I need to pull from different private repositories hosted on GitHub (for now, owned by myself, but this can change in the future). I read the instructions and added the Shippable deployment key to the "Deploy keys" of the first repo, but when I try to do the same for the second repo I get the GutHub error
```
Key is already in use
```

I then removed the Shippable deployment key added to the first repo, and added it to the SSH keys under "Personal settings" --> "SSH and GPG keys", but again CI fails with the [error](https://app.shippable.com/runs/57e25551c8aa940f0088e10b/1/console) 
```
Cloning into 'xxx'...
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/jacopo-chevallard/xxx/'
```

I guess I'm doing something wrong, but couldn't figure it out from reading docs...

**Comment by rageshkrishna** [comment]
Hi @jacopo-chevallard. The deploy keys that get automatically added to your repositories when you enable them on Shippable are "special". We use them only when we clone the specific repository that is being built.

To make your scenario work, you should get the subscription's deploy key from https://app.shippable.com/subscriptions/573b2c812a8192902e205f09/settings/options and add that to your account's personal settings -> ssh keys.

This key is what gets offered when you manually run any `git clone` operations in your build.

You may also need to undo what you tried with the deploy key on the first repo, and move that back from the account to the repository. An easier way to do this is to just hit the "reset" button from the project's settings page. This will generate new keys for your project and set everything up again.

**Comment by jacopo-chevallard** [comment]
Thanks for the rapid answer! This is exactly what I tried to do during my second attempt, and I just repeated it now (after resetting), but I get the same error (``remote: Invalid username or password. fatal: Authentication failed for 'https://github.com/jacopo-chevallard/xxx/'``)

This is the Shippable page from which I took the SSH key

![screen shot 2016-09-21 at 14 13 37](https://cloud.githubusercontent.com/assets/8123595/18710522/fb45e0e8-8005-11e6-9c90-80059c89a184.png)
 
and I added the key here on GitHub

![screen shot 2016-09-21 at 14 10 46](https://cloud.githubusercontent.com/assets/8123595/18710534/0dc951fa-8006-11e6-83ef-79016b129d17.png)

Am I missing something?
